### ISSUE TITLE
Provide way of avoiding the community auditor requirements

### ISSUE THREAD

**Issue by 0xdabbad00** [duo-labs/parliament#86]
The latest release with PolicySentry added for the community auditors adds a lot of additional requirements, including pandas and SQLAlchemy.  This increased the size from 8.4MB (3.1MB of which is the iam definition) to more than 52MB.  The size isn't so much a concern for me as the additional dependencies are.  The inclusion of PolicySentry is only currently used for the rule to identify "Permissions management" APIs, which could instead just be added as a static list of a few dozen privilege names.

**Comment by xen0l** [comment]
@0xdabbad00 we could maybe make community auditors available via extra requirements, which would result in creating 2 versions of parliament. More information here: https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies

**Comment by kmcquade** [comment]
Hmm. How does that require creating 2 versions of Parliament? Perhaps I'm missing something but I'm not gleaning that from the docs. I'm interested in how this would work though.

**Comment by xen0l** [comment]
Sorry, what I meant was 2 variants, e.g. the version with community auditors would be installed like `pip install parliament[community]` and setup.py would need to have similar lines:

```
extra_requires = {
  'community': [
     'dep1', 
     'dep2',
     ...
   ]
}
```

**Comment by 0xdabbad00** [comment]
For the current situation, we can avoid the extra requirements by just adding a static list which seems easiest although it will require some maintenance.  For auditors that do require further requirements, we may need to just keep them in a separate repo that people can install, which was an idea @kmcquade and I had discussed.

**Comment by 0xdabbad00** [comment]
Resolved in #94
